src/index.js

```diff
@@ -27,6 +28,10 @@ export default class Server {
                 reject(new Error('already in use'));
             });
 
+            this.server.on('connection', connection => {
+                this.connections.push(connection);
```

Comment:
Another one little thing. If a connection is closed we should remove it from the list. We can add subscription for `close` event for this
```js
connection.on('close', function() {
      //remove from the list
});
```